app/routes/index.js

```diff
@@ -1,9 +1,39 @@
+require('dotenv').config();
 const express = require('express');
 const router = express.Router();
+const dotenv = require("dotenv");
+const jwt = require('jsonwebtoken');
+
+dotenv.config();
+
+process.env.TOKEN_SECRET;
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-    res.render('index', {title:'OG-YOGA'});
+    res.render('index', {title:'OG-YOGA', error: ""});
+});
+
+router.post('/login', async function(req, res, next) {
+    if(req.body.secret_key !== process.env.TOKEN_SECRET){
```

Comment:
if authentication failed, no need to make a new cookie, just send the error message